app/src/main/java/com/nextcloud/talk/utils/DoNotDisturbUtils.kt

```diff
@@ -20,22 +20,33 @@
 
 package com.nextcloud.talk.utils
 
+import android.annotation.SuppressLint
 import android.app.NotificationManager
 import android.content.Context
 import android.media.AudioManager
 import android.os.Build
 import android.os.Vibrator
+import androidx.annotation.VisibleForTesting
 import com.nextcloud.talk.application.NextcloudTalkApplication
 
 object DoNotDisturbUtils {
-    fun shouldPlaySound(): Boolean {
-        val context = NextcloudTalkApplication.sharedApplication?.applicationContext
+
+    private var buildVersion = Build.VERSION.SDK_INT
+
+    @VisibleForTesting
+    fun setTestingBuildVersion(version: Int) {
+        buildVersion = version
+    }
```

Comment:
This,  is not a "definitive solution". Ideally,

* `DoNotDisturbUtils` would be an injectable class, not an object
* There would be an interface like `BuildInfoProvider` or something, with a default implementation using Build.VERSION.SDK_INT
* Said interface would be injected into `DoNotDisturbUtils` by constructor, using the real implementation in production code, and a fake one with whatever sdk_int we want.